elasticsearch/license.md

```diff
@@ -0,0 +1,2 @@
+View license information for [Elasticsearch](http://www.elasticsearch.org/guide/en/elasticsearch/hadoop/current/license.html) software contained in this image.
```

Comment:
Do you think it might be better to link to https://github.com/elasticsearch/elasticsearch/blob/66b5ed86f7adede8102cd4d979b9f4924e5bd837/LICENSE.txt so that it's more obviously talking about the Elasticsearch license, not `elasticsearch-hadoop`?

Also, this file seems to have a spurious extra newline.